wiki/News_styling_criteria/en.md

```diff
@@ -68,61 +68,34 @@ date: 2017-08-17 03:00:00 +0000
 
 ``title`` must represent the article's full title and name, complete with any appropriate formatting. Markdown formatting is disallowed in this string.
 
-``date`` must represent a combined string displaying a calendar date in ISO 8601 format (2017-08-17), followed by a 24 hour time string (03:00:00), followed by a time offset from UTC (+0000). This date is used to determine the "xx minutes/hours/days ago" display on the website when referring to the publish dates of visible news articles.
+``date`` must represent a combined string displaying a calendar date in ISO 8601 format (2017-08-17), followed by a 24-hour time string (03:00:00), followed by a time offset from UTC (+0000). This date is used to determine the "xx minutes/hours/days ago" display on the website when referring to the publishing dates of visible news articles.
 
 ### Markdown
 
 The osu! wiki uses a partial implementation of GFM (GitHub Flavored Markdown) with the following exceptions:
 
-#### Titles
-
-News articles do not use traditional Markdown titles in any capacity whatsoever.
-
-#### Headers
-
-Header levels 1 (``#``), 2 (``##``), and 3 (``###``) can be used. They have unique identifiers and can be linked directly to. Never use headers to style or format text.
-
-#### Bold
-
-Bolding is reserved for highlighting objects or phrases of importance, be them players, scoring, or whatever else is contextually considered a focus of the article in question.
-
-#### Italics
-
-Italics is used uncommonly to accent phrases or statements of interest within the conversational register. It may also be used to designate names of other creative works, such as the *osu! Coffee Hour* or the *Aspire Beatmapping Contests*.
-
-#### Horizontal bar
-
-Horizontal bars must be used sparingly and only in situations where content needs to be cleanly and definitively separated from the rest of the article.
-
-#### Tables
-
-Markdown tables can be used, but is not recommended for displaying data in news articles. Visualising the data using an image is likely to be more pleasing and more accessible to most users.
-
-### Images
-
-All images linked to news articles must either be hosted on osu! servers (i.e. `assets.ppy.sh`) or placed within and referenced from the `osu-wiki` shared image repository.
-
-News articles that use images have their own folder within ``/wiki/shared/news/`` using the same naming format as their filename, with the exception of the ``.md`` suffix.
-
-Example:
-
-``/wiki/shared/news/2017-08-17-news-article-standards/banner.jpg``
-
-#### Covers
+- Titles
+- Headers
+  - Header levels 1 (``#``), 2 (``##``), and 3 (``###``) can be used. They have unique identifiers and can be linked directly to. Never use headers to style or format text.
```

Comment:
```suggestion
- Headings: Heading level 1 (`#`), corresponding to an article title, must not be used, because it is provided by the front matter.
```